CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 - [Backup NG logs] Handle failed restores (PR [#3648](https://github.com/vatesfr/xen-orchestra/pull/3648))
 - [Self/New VM] Incorrect limit computation [#3658](https://github.com/vatesfr/xen-orchestra/issues/3658)
 - [Plugins] Don't expose credentials in config to users (PR [#3671](https://github.com/vatesfr/xen-orchestra/pull/3671))
+- [Self/New VM] `not enough … available in the set …` error in some cases
```

Comment:
- Is this message clear enough?
- Any related issues?
- Please add the PR reference.

Reply:
- I think it is. This error used to pop up when creating a "New VM" using "Self" and now it doesn't anymore.
- No related issue.
- Done. :angel: